tests/test_flatten.py

```diff
@@ -55,3 +61,14 @@ def test_flatten_pd():
             ]
         )
     )
+
+
+def test_aux_function_is_called():
+    mock_flatten_df = Mock()
+
+    with pytest.MonkeyPatch().context() as m:
+        m.setattr("crossfire._flatten_df", mock_flatten_df)
+
+        flatten(PD_DATA, nested_columns=["contextInfo"])
+
+    mock_flatten_df.assert_called_once()
```

Comment:
Mock the `_flatten_df` to assert if it is being called during the `flatten` function recieves a `DataFrame`.
Realizing now that perhaps the name must be improved, right?
Will update it soon